test/consensus-gateway/setup_consensus_cogateway.js

```diff
@@ -19,10 +19,10 @@ const { AccountProvider } = require('../test_lib/utils.js');
 const Utils = require('../test_lib/utils.js');
 const ConsensusGatewayUtils = require('./utils');
 
-const SpyCoConsensus = artifacts.require('SpyCoConsensus');
+const SpyCoconsensus = artifacts.require('SpyCoconsensus');
 const ConsensusCogateway = artifacts.require('ConsensusCogateway');
 
-contract('CoConsensusgateway::setup', (accounts) => {
+contract('Coconsensusgateway::setup', (accounts) => {
```

Comment:
Rename `Coconsensusgateway` to `ConsensusCogateway `

Reply:
done